eblocker-icapserver/src/test/java/org/eblocker/server/common/data/events/DataSourceEventLoggerTest.java

```diff
@@ -60,41 +55,27 @@ public void eventsAreAdded() throws InterruptedException {
     public void loadTest() throws InterruptedException {
         // Each thread logs 1000 events of a type:
         Thread a = new Thread(() -> {
-            for (int i = 0; i < 1000; i++) {
+            for (int i = 0; i < 3; i++) {
                 logger.log(Events.powerFailure());
-            }
-        });
-        Thread b = new Thread(() -> {
-            for (int i = 0; i < 1000; i++) {
                 logger.log(Events.networkInterfaceDown());
             }
         });
 
         a.start();
-        b.start();
 
         a.join();
-        b.join();
 
-        Thread.sleep(100); // wait for background thread to add events to the datasource
+        while (logger.queueSize() > 0){ // wait for background thread to add events to the datasource
+            Thread.sleep(10);
+        }
 
-        // Verify that the datasource received 1000 events of each type:
-        Stream
-                .of(EventType.POWER_FAILURE,
-                        EventType.NETWORK_INTERFACE_DOWN)
-                .forEach((type) -> {
-                    Mockito.verify(dataSource, Mockito.times(1000)).addEvent(Mockito.argThat(event -> event.getType() == type));
-                });
-    }
 
-    @Test
-    public void testInterruption() throws InterruptedException {
```

Comment:
I could not see that this test was testing production code. I have therefore deleted it.